infra/README.md

```diff
@@ -1,8 +1,25 @@
-## Fetching Debian Bullseye Image and Verifying It
 
-- `./base-image/fetch-debian-image.sh`
+## Environment choice
+This section covers how to set a production-like environment. 
+For most developments, it is better to use the containers instead (see the dev-env repository). The infrastructure environment provides additional services like reverse-proxy (nginx) and monitoring, but is also much less practical.
```

Comment:
dev-env is a directory not a respository

Reply:
indeed I meant directory